packages/grid/_modules_/grid/utils/material-ui-utils.ts

```diff
@@ -44,3 +44,17 @@ export function getScrollbarSize(doc: Document): number {
 
   return scrollbarSize;
 }
+
+export function useId(idOverride) {
```

Comment:
I had to copy this from `code` as the import doesn't work, it returns `undefined` :/

Reply:
The module seems to be exposed as `unstable_useId` in both v4
https://github.com/mui-org/material-ui/blob/91fa0d7bb0c70d5ed98cb1b42aadba4babecf06d/packages/material-ui/src/utils/index.js#L16
and v5

https://github.com/mui-org/material-ui/blob/4793d2d5d6fc8542aa422c0e98261e11649fb994/packages/material-ui/src/utils/index.js#L12

Reply:
Yes I saw it but when you do `import { unstable_useId } from '@material-ui/utils';` the `unstable_useId` is `undefined`.

Reply:
Did you try `@material-ui/core/utils`? I didn't link the utils package.

Reply:
I'll try it

Reply:
Done but I needed to `@ts-ignore` because of a warning that reads that the module doesn't have an exported method `unstable_useId`.